posydon/binary_evol/CE/step_CEE.py

```diff
@@ -397,6 +397,487 @@ def calculate_lambda_CE(self, donor, verbose=False):
+        orbital_period_f = cf.orbital_period_from_separation(
+            separation_postCEE / const.Rsun, mc1_i, mc2_i)
+
+        # Calculate the orbital separation accordingly
+        separation_f = cf.orbital_separation_from_period(orbital_period_f,
+            mc1_f, mc2_f)
+
+        # Check to see if the system has merged
+        merger = cf.check_for_RLO(mc1_f, rc1_f, mc2_f, rc2_f,
+            separation_f, self.CEE_tolerance_err)
+
+        if merger:
+            if verbose:
+                print("system merges within the CEE")
+        else:
+            if verbose:
+                print("system survives CEE, the whole CE is ejected and the "
+                      "new orbital separation for the cores is returned")
+
+        return mc1_f, rc1_f, mc2_f, rc2_f, separation_f, orbital_period_f, merger
+
+    def CEE_core_not_replaced_stableMT(self, donor, mc1_i, rc1_i, 
+                                       donor_type, comp_star, mc2_i, rc2_i, 
+                                       comp_type, double_CE, 
+                                       separation_postCEE, verbose=False):
+        """Calculate the post-common-envelope parameters upon exiting a CEE.
+
+        This prescription assumes the he_core_mass/radius (or 
+        co_core_mass/radius for CEE of stripped_He*) stays as preCEE and after 
+        a succesful ejection using the alpha-lambda prescription, we assume an 
+        instantaneous stableMT phase (non-conservative, with mass lost from 
+        accretor) from the donor (or from donor and simultaneously the accretor
+        at double_CE), taking away the extra "core" mass as defined by the core
+        boundary used for CEE (based on core_definition_H/He_fraction).
+
+        Parameters
+        ----------
+        donor : SingleStar object
+            The donor star
+        mc1_i : float
+            core mass of the donor (in Msun)
+        rc1_i : float
+            core radius of the donor (in Rsun)
+        comp_star : SingleStar object
+            The companion star
+        mc2_i : float
+            core mass of the companion (in Msun)
+        rc2_i : float
+            core radius of the companion (in Rsun)
+        separation_postCEE : float
+            binary's separation upon exiting the CEE (in cm)
+        verbose : bool
+            In case we want information about the CEE.
+
+        Returns
+        -------
+        m1c_f: float
+            donor mass after the CE (in Msun)
+        r1c_f: float
+            donor radius after the CE (in Rsun)
+        m2c_f: float
+            companion mass after the CE (in Msun)
+        r2c_f: float
+            companion radius after the CE (in Rsun)
+        separation_f : float
+            binary's final separation upon exiting the CE (in Rsun)
+        orbital_period_f : float
+            binary's final orbital period upon exiting the CE (in days)
+        merger : bool
+            whether the binary merged in the CE
+        """
```

Comment:
Capture double CE case and send it to the wind loss prescription.
```suggestion
        """
        if double_CE:
            Pwarn("A double CE cannot have a stable mass transfer afterwards "
                  "as both cores are donors, switch to losing the mass as "
                  "wind.", "ReplaceValueWarning")
            return self.CEE_core_not_replaced_windloss(donor, mc1_i, rc1_i,
                                                       donor_type, comp_star,
                                                       mc2_i, rc2_i, comp_type,
                                                       double_CE,
                                                       separation_postCEE,
                                                       verbose)
```